packages/@aws-cdk/aws-cloudwatch-actions/README.md

```diff
@@ -25,4 +25,19 @@ alarm.addAlarmAction(
 );
 ```
 
+## SSM OpsCenter Action Example
+
+```ts
+import * as cw from "@aws-cdk/aws-cloudwatch";
+
+let alarm: cw.Alarm;
+// Create an OpsItem with specific severity and category when alarm triggers
+alarm.addAlarmAction(
+  new SsmAction(
+    OpsItemSeverity.CRITICAL,
+    OpsItemCategory.PERFORMANCE // category is optional
+  )
+);
+```
```

Comment:
Now that strict mode is turned on for README examples, this one is failing compilation:

```suggestion
```ts
declare const alarm: cloudwatch.Alarm;
// Create an OpsItem with specific severity and category when alarm triggers
alarm.addAlarmAction(
  new actions.SsmAction(
    actions.OpsItemSeverity.CRITICAL,
    actions.OpsItemCategory.PERFORMANCE // category is optional
  )
);
```
```